src/pages/Defi/components/EligibleSlider.tsx

```diff
@@ -17,10 +17,16 @@ type EligibleSliderProps = {
 
 export const EligibleSlider: React.FC<EligibleSliderProps> = ({ slidesToShow = 4, ...rest }) => {
   const eligibleOpportunities = useAppSelector(selectAggregatedEarnUserStakingEligibleOpportunities)
-  const renderEligibleCards = useMemo(() => {
+
+  const featuredOpportunities = useMemo(() => {
     // opportunities with 1% APY or more
     const filteredEligibleOpportunities = eligibleOpportunities
-      .filter(o => bnOrZero(o.tvl).gt(50000) && (!o.apy || bn(o.apy).gte(0.01)))
+      .filter(
+        o =>
+          bnOrZero(o.tvl).gt(50000) &&
+          (!o.apy || bn(o.apy).gte(0.01)) &&
+          o.provider !== DefiProvider.ThorchainSavers,
```

Comment:
ditto remark re: flag/mimir vs. hardcoded magic

Reply:
Done in [59f35c1](https://github.com/shapeshift/web/pull/8681/commits/59f35c1b7b628ff50975e1d958516465cee23df4)